Resources/Prototypes/DeltaV/Entities/Objects/Misc/identification_cards.yml

```diff
@@ -0,0 +1,38 @@
+- type: entity
+  parent: IDCardStandard
+  id: CJIDCard
+  name: chief justice ID card
+  components:
+    - type: PresetIdCard
+      job: ChiefJustice
+    - type: Sprite
+      layers:
+        - state: default
+        - sprite: DeltaV/Objects/Misc/id_cards.rsi
+          state: idchiefjustice
```

Comment:
Worth considering if this could be a steal objective.

Reply:
It's less valuable than most of the head's ID cards and none of them have steal objectives. Security IDs would be more valuable